src/aiida_quantumespresso/parsers/open_grid.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from aiida.common import NotExistent
+from aiida.orm import KpointsData
+
+from aiida_quantumespresso.parsers.base import Parser
+from aiida_quantumespresso.parsers.parse_raw.base import parse_output_base
+
+
+class OpenGridParser(Parser):
+    """``Parser`` implementation for the ``OpenGridCalculation`` calculation job class."""
+
+    def parse(self, **kwargs):
+        """Parse the retrieved files of a completed ``OpenGridCalculation`` into output nodes."""
+        try:
+            out_folder = self.retrieved
+        except NotExistent:
+            return self.exit(self.exit_codes.ERROR_NO_RETRIEVED_FOLDER)
+
+        try:
+            filename_stdout = self.node.get_option('output_filename')
+            with out_folder.open(filename_stdout, 'r') as handle:
+                out_file = handle.read()
+        except OSError:
+            return self.exit(self.exit_codes.ERROR_OUTPUT_STDOUT_READ)
+
+        parsed_data, logs = parse_output_base(out_file, codename='OPEN_GRID')
```

Comment:
Since the returned `parsed_data` here also returns the QE version, perhaps we can also add a warning to the logs here in case the QE version is < 7.0?

On another note, most of the other `NamelistsCalculation`s also output the version and walltime to an `output_parameters` output. Might be nice to add these here as well for consistency, and in case someone e.g. wants to get the full wall time of a work chain.

Reply:
For QE < 7.0, the `open_grid.x` wouldn't run at all so probably the parser will output empty results.
But the wall time info is indeed useful, added in https://github.com/aiidateam/aiida-quantumespresso/pull/714/commits/a4689cfe23010a91cff766d16ef7939d13afaffe